src/main/webapp/resources.xml

```diff
@@ -21,8 +21,8 @@
 -->
 <resources xmlns="http://www.jasig.org/uportal/web/skin" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
-    <js included="plain" resource="true">/rs/jquery/1.10.2/jquery-1.10.2.js</js>
-    <js included="aggregated" resource="true">/rs/jquery/1.10.2/jquery-1.10.2.min.js</js>
+    <js included="plain" resource="true">/rs/jquery/1.11.0/jquery-1.11.0.js</js>
+    <js included="aggregated" resource="true">/rs/jquery/1.11.0/jquery-1.11.0.min.js</js>
```

Comment:
Note that `jquery-1.11.0.min.js` may need to be added to resource server.
https://github.com/Jasig/resource-server/tree/master/resource-server-content/src/main/webapp/rs/jquery/1.11.0